<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>1856</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="text_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>30.06.2016</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>13:51</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_temp">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>10</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>48</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>20Â°</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_min_temp">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>140</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_max_temp">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>140</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_todo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1460</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Today's Tasks:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_tasks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>1530</y>
     <width>331</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="text_station">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>1460</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Parzivalplatz</string>
   </property>
  </widget>
  <widget class="QLabel" name="draw_mvg1">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>1530</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_dest_1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>1530</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_mvg_time1">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>1530</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 min</string>
   </property>
  </widget>
  <widget class="QLabel" name="draw_mvg2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>1590</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="draw_mvg3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>1650</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="draw_mvg4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>1710</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="draw_nvg5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>1770</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_dest2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>1590</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_dest3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>1650</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_dest4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>1710</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_dest5">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>1770</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_mvg_time2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>1590</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 min</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_mvg_time3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>1650</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 min</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_mvg_time4">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>1710</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 min</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_mvg_time5">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>1770</y>
     <width>55</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 min</string>
   </property>
  </widget>
  <widget class="QLabel" name="draw_weather_logo">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>181</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>48</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>20Â°</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
